server/src/visitors/YmlFunctionVisitor.ts

```diff
@@ -106,20 +117,12 @@ export class YmlFunctionVisitor extends YmlBaseVisitor {
 
     /**
      * Check if the provided function name is the name of a class method instanciation.
-     * If the function name is the name of a class method instanciation:
-     * - it follows the format “className” + “::” + “methodName”
-     * - it already exists as a completion item thanks to `YmlClassVisitor`
+     * If the function name is the name of a class method instanciation if
+     * it follows the format “className” + “::” + “methodName”
      * @param fullName The name of a function.
      * @returns `true` iff the provided functionName is a class method instanciation.
      */
     private isMethodInstanciation(fullName: string): boolean {
-        let className: string;
-        let methodName: string;
-        const functionNameSubParts = fullName.split('::');
-        if (functionNameSubParts.length > 1) {
-            methodName = functionNameSubParts.pop();
-            className = fullName.replace(RegExp(`::${methodName}$`, 'u'), '');
-        }
-        return className && this.completionProvider.getItem(`id_${className}_${fullName}`);
+        return fullName.split('::').length > 1;
```

Comment:
I'm not sure this is the right method just to know how much times you've got the sequence "::" in the string. This one create a tab of string, maybe you can find something more light.

Reply:
I propose this:
```suggestion
        // At least one character and “::”.
        return fullName.indexOf('::') > 0;
```

Reply:
Ok for me :-)